### ISSUE TITLE
CI failing due to dependency issues

### ISSUE THREAD

**Issue by elichad** [ResearchObject/packaging_data_with_ro-crate#2]
"Website build" job is now failing:
```
Run gem install github-pages bundler kramdown kramdown-parser-gfm
ERROR:  Error installing github-pages:
	The last version of nokogiri (>= 1.13.6, < 2.0) to support your Ruby & RubyGems was 1.15.6. Try installing it with `gem install nokogiri -v 1.15.6` and then running the current command again
	nokogiri requires Ruby version >= 3.0, < 3.4.dev. The current ruby version is 2.7.8.225.
ERROR:  Error installing bundler:
	The last version of bundler (>= 0) to support your Ruby & RubyGems was 2.4.22. Try installing it with `gem install bundler -v 2.4.22`
	bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.8.225.
Successfully installed unicode-display_width-1.8.0
Successfully installed terminal-table-1.8.0
Successfully installed kramdown-2.4.0
Successfully installed kramdown-parser-gfm-1.1.0
2 gems installed
```

The install and `make lesson-check-all` work on my machine with Ruby 3.3 (CI uses 2.7), so changing that might work - though the long-term solution is to upgrade to use the Workbench, I think.

**Comment by stain** [comment]
@OliverWoolland has worked on the new template:

> This template is deprecated. Please use one of the template lesson repositories that uses The Carpentries Workbench:
>     A template for lessons using standard Markdown: https://github.com/carpentries/workbench-template-md
>     A template for lessons using R Markdown (recommended for lessons that will include example R code blocks): https://github.com/carpentries/workbench-template-rmd

**Comment by elichad** [comment]
See https://carpentries.github.io/sandpaper-docs/instructor/migrating-from-styles.html for the latest migration guidelines

**Comment by douglowe** [comment]
I've finished the first stage of migrating the material, currently on my personal github: https://github.com/douglowe/packaging_data_with_ro-crate

Website: https://douglowe.github.io/packaging_data_with_ro-crate/index.html

If it looks reasonable I'll replace the contents of the current repo with this one.

**Comment by douglowe** [comment]
@elichad, @OliverWoolland - could you check this material over, make sure it's an okay representation of the original?

**Comment by elichad** [comment]
The migration of the content looks good to me (I only skimmed it, but didn't spot any glaring formatting issues). 

I found a couple of other small issues while poking around:

- the instructor view of the [summary & schedule page](https://douglowe.github.io/packaging_data_with_ro-crate/instructor/index.html) has some `::::::::::::` visible in the schedule that should be removed
- "Instructor Notes" link at the top gives a 404 - but there is another Instructor Notes page under "More" - think this file just needs renaming from `instructors-notes` to `instructor-notes`
- Under ["Key Points"](https://douglowe.github.io/packaging_data_with_ro-crate/instructor/key-points.html) the first heading is shown twice as "IntroductionIntroduction" - no idea how to fix this one!

Not sure if it is easier to migrate the repo and then fix those, or fix and then migrate.

**Comment by douglowe** [comment]
Unfortunately I think the `::::::::::::::::` bars are a problem with the template. Other course material we've written has this too (see, for example, https://uomresearchit.github.io/docker-introduction/instructor/index.html)

**Comment by douglowe** [comment]
The IntroductionIntroduction issue was caused by the first header in that section being the wrong level (#, not ##). I've fixed this now.

**Comment by douglowe** [comment]
I've renamed to instructor-notes.md - so the Instructor Notes link works now (well, displays the FIXME page).

**Comment by douglowe** [comment]
As the one outstanding issue (other than more content being needed) looks like a template problem, rather than our code, shall we migrate the repo now?

**Comment by elichad** [comment]
Yes, let's migrate - but make a tag with the current `gh-pages` branch first in case we need to revert

**Comment by douglowe** [comment]
I'll set that up now

**Comment by douglowe** [comment]
New website is up: https://www.researchobject.org/packaging_data_with_ro-crate/

I've tagged the old gh-page branch as v0.1-original. I have also moved that branch to gh-page-old, so that we have that record too.

**Comment by elichad** [comment]
Excellent, thank you!

I found another couple of small things to fix when I looked again, but I'll make a new issue for those
